src/main/java/graphql/language/NullValue.java

```diff
@@ -25,6 +25,20 @@ protected NullValue(SourceLocation sourceLocation, List<Comment> comments, Ignor
         return Collections.emptyList();
     }
 
+    @Override
+    public NodeChildrenContainer getNamedChildren() {
+        return NodeChildrenContainer.newNodeChildrenContainer().build();
+    }
+
+    @Override
+    public NullValue withNewChildren(NodeChildrenContainer newChildren) {
+        if (!newChildren.isEmpty()) {
+            throw new IllegalArgumentException("Cannot pass non-empty newChildren to Node that doesn't hold children");
+        }
+
+        return this;
+    }
```

Comment:
Yeah - defintely common method if this is needed

    @Override
    public NullValue withNewChildren(NodeChildrenContainer newChildren) {
           return NodeChildrenContainer.assertNotEmpty(newChildren);
    }

Reply:
refactored it